plugins/common/shim/example/cmd/main.go

```diff
@@ -46,15 +46,14 @@ func main() {
 	// otherwise, follow what the config asks for.
 	// Check for settings from a config toml file,
 	// (or just use whatever plugins were imported above)
-	err = shimLayer.LoadConfig(configFile)
-	if err != nil {
-		fmt.Fprintf(os.Stderr, "Err loading input: %s\n", err)
+	if err = shimLayer.LoadConfig(configFile); err != nil {
+		_, _ = fmt.Fprintf(os.Stderr, "Err loading input: %s\n", err)
 		os.Exit(1)
 	}
 
-	// run a single plugin until stdin closes or we receive a termination signal
-	if err := shimLayer.Run(*pollInterval); err != nil {
-		fmt.Fprintf(os.Stderr, "Err: %s\n", err)
+	// run a single plugin until stdin closes, or we receive a termination signal
+	if err = shimLayer.Run(*pollInterval); err != nil {
+		_, _ = fmt.Fprintf(os.Stderr, "Err: %s\n", err)
```

Comment:
Could we add `fmt.Fprintf` to the `unhandled-error` linter exception list: https://github.com/influxdata/telegraf/blob/master/.golangci.yml#L72 to avoid adding the blank return values

Reply:
@sspaink 
I've added: `fmt.Fprintf` ,`fmt.Fprint` and `fmt.Fprintln`.